applications/Archisinal.md

```diff
@@ -0,0 +1,310 @@
+
+**Artem Smolyarenko** - UI/UX designer with over 3 years experience. He specializes in brand development in all aspects and have an excellent understanding of typography, color psychology, as well as grid, web and mobile principles. Artem worked on different types of products with startups and corporate teams from around the world. Skills: UI/UX design, product design, web and mobile app design. Tech stack & tools: Figma, Adobe Photoshop, Adobe Illustrator, Adobe XD.
+
+### Team Code Repos
+
+- **Archisinal** [https://github.com/Archisinal](https://github.com/Archisinal)
+- **Danylo Semirazov** - [github.com/sventime](https://github.com/sventime), [gitlab.com/asylum-space](https://gitlab.com/asylum-space). Contributions Github: [Openbrush Contracts](https://github.com/Supercolony-net/openbrush-contracts), [Grants-Program: Asylum](https://github.com/w3f/Grants-Program/blob/master/applications/asylum.md),[Grants-Program: Asylum follow-up](https://github.com/w3f/Grants-Program/blob/master/applications/asylum_follow_up_1.md), [Asylum Grant Milestone Delivery](https://github.com/w3f/Grant-Milestone-Delivery/pull/563),[Astar](https://github.com/AstarNetwork/Astar/pull/593), [Workshop: Blockchain from scratch on Rust](https://github.com/sventime/blockchain_workshop_oct_16)
+- **Oleh Kulykovskyi**  - [github.com/blindbeat](http://github.com/blindbeat), [gitlab.com/asylum-space](https://gitlab.com/asylum-space).
+
+### Team LinkedIn Profiles (if available)
+
+- **Tom Humber** https://www.linkedin.com/in/tom-humber-b65888118
+- **Danylo Semirazov** - [linkedin.com/in/danylosemirazov/](https://www.linkedin.com/in/danylosemirazov/)
+- **Oleh KulyKovskyi**  - [linkedin.com/in/oleh-kulykovskyi-329523234/](http://www.linkedin.com/in/oleh-kulykovskyi-329523234/)
+
+
+## Development Status :open_book:
+
+- [Medium](https://medium.com/@archisinal)
+- [Twitter](https://twitter.com/archisinal)
+- [Instagram](https://www.instagram.com/archisinal.io/)
+
+## Development Roadmap :nut_and_bolt: ##
+
+### Overview
+
+The development roadmap for Archisinal includes multiple milestones to ensure the successful launch and operation of the platform.
+
+### **Overview**
+
+- **Total Estimated Duration:**
+    - MVP: 2.5 months
+    - Post-MVP: 9 months
+    - Post-Launch: 10 months
+- **Full-Time Equivalent (FTE): 6**
+- **Total Costs:**
+    - MVP: $60,000
+
+## MVP (brief discussion)
+
+### **Milestone 1: Smart Contract Development**
+
+- **Estimated duration:** 1 month
+- **FTE:** 3
+- **Costs:** 24,000 USD
```

Comment:
This cost doesn't quite add up for me. 

You have two developers and one designer. Of these, only one has Rust experience. Let's assume that you're going to get the front-end developer writing Rust contracts, that's still only two FTEs that are needed for this smart-contract only milestone.

Reply:
Thank you, @HCastano, for your valid input. You're absolutely right that our UI/UX designer won't be involved in the 1st milestone. Let me provide more details about the responsibilities of our team members for the 1st milestone:

1. As a Rust engineer, my primary focus will be on the smart contracts part.
2. Our front-end engineer, Oleg, has prior experience in writing and testing smart contracts, and his main focus for this milestone will be on TypeScript integration tests.
3. Tom (@Archisinal) will be engaged in creating the architecture and will also be involved in the review process.

For the 2nd milestone, our development team will consist of me (for the back-end part), Oleg (for the front-end), and Artem (UI/UX design).

Reply:
I hope the Rust code also has some integration tests and you don't rely on just the TS ones 😉 

Alright, thanks for clarifying 👍